crates/aggchain-proof-core/src/bridge/mod.rs

```diff
@@ -328,15 +328,13 @@ impl BridgeConstraintsInput {
         Ok(bridge_address.into())
     }
 
-    /// Verify that the claimed global indexes minus the unset global indexes
-    /// are equal to the Constrained global indexes.
-    fn verify_constrained_global_indices(&self) -> Result<(), BridgeConstraintsError> {
-        // Check if the filtered claimed global indices are equal to the constrained
-        // global indices.
+    /// Verify that the claims filtered with the global indexes that got
+    /// unclaimed are equal to commited imported bridge exits.
+    pub fn verify_constrained_claims(&self) -> Result<(), BridgeConstraintsError> {
         let constrained_claims = ImportedBridgeExitCommitmentValues {
             claims: filter_values(
-                &self.bridge_witness.global_indices_unset,
-                &self.bridge_witness.bridge_exits_claimed,
+                &self.bridge_witness.unset_claims, // Vec<U256> of unclaimed global indexes
+                &self.bridge_witness.bridge_exits_claimed, // Vec<GlobalIndexWithLeafHash>
```

Comment:
Nit: This seems like a fairly useless comment
```suggestion
                &self.bridge_witness.bridge_exits_claimed,
```